contracts/PoolRouter.sol

```diff
@@ -40,43 +41,58 @@ contract PoolRouter {
     function batch(
         PoolDataTypes.Operation[] calldata operations,
         bytes[] calldata data
-    ) external payable {
+    ) external payable returns(bytes[] memory resultTypes, bytes[] memory resultValues) {
```

Comment:
I'm concerned that if we follow this pattern with Ladle it will be too much bytecode and we will have to reduce the optimization runs again. Do we really need to return the type of each return value? The caller knows each action of the batch, and should be able to decode the return values without help.